<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayList</class>
 <widget class="QWidget" name="PlayList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PlayList</string>
  </property>
  <widget class="QWidget" name="wgt_playList" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>490</width>
     <height>450</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#pb_playList_close{
    border-image: url(:/common/skins/common/closePressed.png);
}

QPushButton#pb_playList_close:pressed{
    border-image: url(:/common/skins/common/closeNormal.png);
}

QListWidget#lw_songsList{
    color: purple;
    font: bold;
    font-family: &quot;楷体&quot;;
    font-size: 20px;
}

QListWidget#lw_songsList::item{
    min-height: 30px;
    max-height: 30px;
    background-color: transparent;
}

QListWidget#lw_songsList::item:hover{
    color: red;
}

QListWidget#lw_songsList::item:selected{
    color: blue;
}
</string>
   </property>
   <widget class="QListWidget" name="lw_songsList">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>300</width>
      <height>375</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLabel" name="lb_playList">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>116</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>播放列表</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_playList_close">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>20</y>
      <width>17</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
